### PR TITLE
first version of UnsubscribeShopfloorInformation

### LINKED ISSUE #232: [New Model]: SIS ShopfloorInformationService: SISUnsubscribe
## Model Description
The shopfloor information results contains the forecasting information exchanged between the Modular Production and a second party (customer, logistical etc.).
The forecasting information gives an overview about the current processing state of a product and the estimated production date.
This information will be exchanged between the partners either on request, cyclic with a fixed cycle or on notification, if a forecast is changing.
In Case of cyclic and notification Exchange, it is necessary to unsubscribe from the notification, which is dealt in this 

## Example Data
SMC|Property | mand./opt. | Semantic ID | Remark
--|-- | -- | -- | --
**header**|RequestID|mandatory||	
||senderBpn CXPartnerID | mandatory| identification of the sender|
||recipientBpn | mandatory|identification of the recipient|
||context| mandatory| Information about the context the message should be considered in, e.g. "MP-Request"|
||messageId| mandatory| unique ID for message will be used as requestID for the following communication|
||sentDateTime| mandatory| time of request|
||version| mandatory| of meta model used for compatability|
||relatedMessageId| mandatory|UUID of request messageId for identification of related messages|
**body**

    

## MS1 Criteria

- [ ] The proposed aspect model does not exist already
- [ ] The proposed aspect model does not extend an existing aspect but introduces completely new functionality
- [ ] The proposal references a Catena-X use case
- [ ] Relevant standards are mentioned/linked
- [ ] A first draft of the data structure is provided in an example

### PR DESCRIPTION
**Pull request by wehrstedt-jan**
## Description

 -->

Closes #232

requires use of header https://github.com/eclipse-tractusx/sldt-semantic-models/pull/227 

## MS2 Criteria
(to be filled out by PR reviewer)
- [ ] the model **validates** with the BAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar bamm-cli.jar -i \<path-to-aspect-model\> -v ). The  BAMM CLI is available [here](https://openmanufacturingplatform.github.io/sds-documentation/sds-developer-guide/dev-snapshot/tooling-guide/bamm-cli.html) and in [GitHub](https://github.com/OpenManufacturingPlatform/sds-sdk/releases)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "preferred name" and "description"** in English language. The description must be comprehensible. It is not required to write full sentences but style should be consistent over the whole model
- [x] Property and the referenced Characteristic should not have the same name
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [ ] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the BAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [ ] when relying on **external standards**, they are referenced through a **"see"** element
- [ ] all properties with an [simple type](https://openmanufacturingplatform.github.io/sds-documentation/bamm-specification/v1.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [ ] generated json schema validates against example json payload
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [ ] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [ ] All required reviewers have approved this PR (see reviewers section)
- [ ] The new aspect (version) will be implemented by at least one data provider
- [ ] The new aspect (version) will be consumed by at least one data consumer
- [ ] There exists valid test data
- [ ] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [ ] The model has at least version '1.0.0'
- [ ] The release date in the Release Note is set to the date of the MS3 approval

### CODE REVIEW

File: io.catenax.mp_standard_unsubscribe/1.0.0/UnsubscribeShopfloorInformation.ttl [hunk 1]
```diff
@@ -0,0 +1,42 @@
+##########################################################################################
+# Copyright (c) 2023 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e. V.
+# Copyright (c) 2023 Siemens AG
+#
```
Comment (bs-jokri) [inline io.catenax.mp_standard_unsubscribe/1.0.0/UnsubscribeShopfloorInformation.ttl:4]: # Copyright (c) 2023 Contributors to the Eclipse Foundation

File: io.catenax.mp_standard_unsubscribe/1.0.0/UnsubscribeShopfloorInformation.ttl [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.mp_standard_unsubscribe:1.0.0#> .
+@prefix ext-header: <urn:samm:io.catenax.messaging_header:1.0.0#> .
```
Comment (bs-jokri) [inline io.catenax.mp_standard_unsubscribe/1.0.0/UnsubscribeShopfloorInformation.ttl:24]: if renaming of messaging_header model is required do not forget to adjust here

File: io.catenax.mp_standard_unsubscribe/1.0.0/UnsubscribeShopfloorInformation.ttl [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+:UnsubscribeShopfloorInformation a samm:Aspect ;
+   samm:preferredName "UnsubscribeShopfloorInformation"@en ;
```
Comment (bs-jokri) [inline io.catenax.mp_standard_unsubscribe/1.0.0/UnsubscribeShopfloorInformation.ttl:27]: please use normal orthography and proper noun. For whole model. Will not comment any further occasions

File: io.catenax.mp_standard_unsubscribe/RELEASE_NOTES.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Changelog
+
+All notable changes to this model will be documented in this file.
+
+## [1.0.0]
```
Comment (bs-jokri) [inline io.catenax.mp_standard_unsubscribe/RELEASE_NOTES.md:5]: add date of MS3 approval as soon as it is available

**Review by bs-jokri — CHANGES_REQUESTED** [review]
please check comments

**Comment by bs-jokri** [comment]
@wehrstedt-jan I understood that this model is not required anymore. If so, could you please close the PR then.

**Comment by wehrstedt-jan** [comment]
not required anymore

**Pull request closed without merging**